### DISCUSSION TITLE
1.8.5版本版本bug

### DISCUSSION THREAD

**Discussion by kevin-hao** [hhyo/Archery#2085] [Q&A]
1、WSGIRequest无法获取user
报错：
<img width="1097" alt="图片" src="https://user-images.githubusercontent.com/14213577/225847780-64c262f0-0cdb-40bd-99f4-fbc705f6e45c.png">

2、decode报错
在查询、上线，选择数据库报错'str' object has no attribute 'decode'
<img width="904" alt="图片" src="https://user-images.githubusercontent.com/14213577/225847690-2f11a7ce-3e85-49e8-8aec-187c918fb13a.png">

**Comment by hhyo** [comment]
请尝试使用新版本
